install

```diff
@@ -33,8 +33,7 @@ if [ "$GH_COMMIT" != "" ]; then
 else
   echo "==> Downloading the latest version of Dev Tools"
   curl -s -L "https://api.github.com/repos/$GH_ORG/$GH_PROJECT/releases/latest" \
-    | grep '"tag_name":' \
-    | sed -E 's/.*"([^"]+)".*/\1/' \
+    | sed -E 's/.*tag_name":\s*"([^,]*)",.*/\1/'
```

Comment:
@tim-yao just tried this on mine. The curl returns everything in multiple lines for me but your command doesn't return the version as expected. Let's put this on hold and have a better look at it.